packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts

```diff
@@ -163,7 +163,7 @@ describe('Liveness Machine', () => {
     jest.advanceTimersToNextTimer(); // waitForSessionInformation
     await flushPromises(); // detectFaceDistanceBeforeRecording
     jest.advanceTimersToNextTimer(); // checkFaceDistanceBeforeRecording
-    service.send({ type: 'START_RECORDING' });
+    // await flushPromises();
```

Comment:
Assuming this is unintentional? 😅

Reply:
whoops haha, this is when i was testing stuff :P will fix